Bug test_get_crimes_by_crime_nature

## Describe the bug
The test `test_get_crimes_by_crime_nature` inside `test_crime.py` is failing when mongo has no data.
This test need to be fixed and remove the hardcoded positions inside an array.

The error stacktrace:
```
=================================== FAILURES ===================================
__________________ TestCrime.test_get_crimes_by_crime_nature ___________________

self = <test_crime.TestCrime testMethod=test_get_crimes_by_crime_nature>

    def test_get_crimes_by_crime_nature(self):
        """
        Testing get crimes by the crime nature
        """
        result, status = controller.get_all_crimes(None, "Roubo a Transeunte")
        self.assertEqual(status, 200)
>       self.assertEqual(result[0]['cities'][9]['Gama'][0]['crime_nature'],
                        "Roubo a Transeunte")
E       IndexError: list index out of range

tests/test_crime.py:56: IndexError
=========================== short test summary info ============================
FAILED tests/test_crime.py::TestCrime::test_get_crimes_by_crime_nature - Inde...
========================= 1 failed, 2 passed in 0.26s ==========================
```

## To Reproduce
Steps to reproduce the behavior:
1. Go to root folter of the project.
2. Run ```docker-compose run api pytest```
4. See the error

## Expected behavior
All tests passed successfully.

## Additional context
You should remove the commend to the lines inside `test_get_crimes_by_crime_nature` method.

--- a/src/tests/test_crime.py
+++ b/src/tests/test_crime.py
@@ -2,6 +2,7 @@ import unittest
 
 from controllers import crime as controller
 from database.db import db
+from random import randint
 
 
 class TestCrime(unittest.TestCase):
@@ -53,5 +54,13 @@ class TestCrime(unittest.TestCase):
         """
         result, status = controller.get_all_crimes(None, "Roubo a Transeunte")
         self.assertEqual(status, 200)
-        # self.assertEqual(result[0]['cities'][9]['Gama'][0]['crime_nature'],
-        #                 "Roubo a Transeunte")
+        if self.db_df_len == 0 and self.db_sp_len == 0:
+            self.assertEqual(result, [])
+        else:
+            index = randint(0, 5)
+            cities = [
+                city for city in result[0]['cities'][index].keys()]
+            for city in cities:
+                self.assertEqual(
+                    result[0]['cities'][index][city][0]['crime_nature'],
+                    'Roubo a Transeunte')

